商品詳細表示機能実装

ynghr1226 opened the pull request:
## What
商品詳細表示機能実装

## Why
商品詳細表示機能実装のため

ynghr1226 committed:
商品詳細表示機能実装

i-date commented on app/views/items/show.html.erb:
```diff
@@ -7,7 +7,7 @@
       <%= "商品名" %>
```
7行目に「商品名」を表示させるためのコードがありますので、こちらも「商品名」が表示できるように修正しましょう。

i-date reviewed (commented):
レビュー依頼ありがとうございます！
コメントいたしましたので、ご確認をお願いいたします。

i-date commented:
また恐れ入りますが、上記修正後に、実装いただいた機能動作や画面を確認させていただきたいので、プルリクに下記の項目を確認できるキャプチャ動画（のURL）を載せていただき、再度プルリクエストをご提出いただければと存じます。
- ログイン状態の出品者のみ、「編集・削除ボタン」が表示される動画
- ログイン状態の出品者以外のユーザーのみ、「購入画面に進むボタン」が表示される動画
- ログアウト状態のユーザーには、「編集・削除・購入画面に進むボタン」が表示されない動画

お手数をおかけいたしますが、よろしくお願いいたします。

ynghr1226 committed:
修正_1

ログアウト状態
https://gyazo.com/a793f6425c5de0e6dfcafb52027cc5aa

ログインユーザー＝出品ユーザー
https://gyazo.com/457ca114051ffbba2111bcf6ebf353a5

ログインユーザー＝出品ユーザー以外
https://gyazo.com/01e788d01f3bb66dc94026db93396b1e

iy06 reviewed (commented):
レビュー依頼ありがとうございます。
LGTMとさせていただきます。実装お疲れ様でした。

マージ後、ローカルのmasterブランチに忘れずにプルしましょう。
その後、新しいブランチを作成し次の実装に取り組んでいただけますと幸いです。
引き続き実装頑張っていきましょう。
